<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="horizontalSlider_CurFreq">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>240</y>
      <width>768</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>80</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_CurFreq">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>230</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>80</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>230</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>80</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>MHz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PageUp">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>350</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>∧</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PageDown">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>460</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>∨</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_NextSatation">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>110</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>＞</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PrevStation">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>＜</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_FM">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>320</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AM">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>590</y>
      <width>320</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ScanSatation">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>590</y>
      <width>320</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>搜台</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Power">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开关</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_FreqMin">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>87.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_FreqMax">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>280</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>108.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>0</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_VolSetting">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>590</y>
      <width>320</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Station1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>220</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0.000MHz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Station2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>220</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0.000MHz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Station4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>220</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0.000MHz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Station5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>220</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0.000MHz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Station3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>370</y>
      <width>220</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0.000MHz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Station6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>480</y>
      <width>220</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0.000MHz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RadioUp">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>220</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>＞</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RadioDown">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>＜</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_vol">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>430</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-60</number>
    </property>
    <property name="maximum">
     <number>24</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
